hashing/te-tag-query-python/all_in_one_lib/content_type/photo.py

```diff
@@ -0,0 +1,16 @@
+##!/usr/bin/env python
+
+"""
+Wrapper around the video content type.
+"""
+import typing as t
+
+from ..signal_type import md5
+from ..signal_type.base import SignalType
+from .base import ContentType
+
+
+class PhotoContent(ContentType):
+    @classmethod
+    def get_signal_types(cls) -> t.List[t.Type[SignalType]]:
+        return [md5.PhotoMD5Signal]
```

Comment:
are we going with photo md5 in any significant way or is this just for proof-of-concept? (i'm not aware of good recall for photo md5.)

Reply:
I included it as a proof of concept, there's some photo MD5s for a collaboration that we sent on request, but this needs a note to use PDQ instead (as soon as we provide PDQ...)